src/xperm_rules.md

```diff
@@ -74,6 +75,26 @@ Conditional Policy Statements
 | ----------------------- | ----------------------- | ----------------------- |
 | No                      | No                      | No                      |
 
+### Extended Permission Evaluation
+
+Extended permission rules are evaluated as follows:
+
+* If no extended permissions are defined, only the resource-level policy is
+  considered.
+
+* If an extended permission rule is defined, the policy is first evaluated
+  according to the high-level resource policy. For example:
+
+  * If an *allowxperm* rule is defined, extended permissions will only be
+    granted if *allow* is granted to the resource. 
+
+  * If an *auditallowxperm* rule is defined, extended audit permissions will only 
+    be granted if *auditallow* is granted to the resource. 
+
+* If any extended permission rule is defined, the resource and operation are fully
+  evaluated according to extended access rules. All undefined permissions within 
```

Comment:
I think "unspecified" would be clearer than "undefined" here.  It might be worth it at the end to take an extra sentence to reiterate that xperms are deny-by-default-allow-by-exception.